Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:8.5.0-alpine
+
+MAINTAINER Mr. David Sibobbabley
+
+RUN npm install -g dev-ringer
+
+CMD [ "ringer", "-H", "/docs/examples/example.har.json", "-o", "/docs/examples/example.drp.json" ]
```

Comment:
since `/docs/examples/example.drp.json` already exists, this should be `CMD [ "ringer", "-f", "/docs/examples/example.drp.json" ]